api/server/v1/marshaler.go

```diff
@@ -643,6 +643,105 @@ func (x *SubscribeResponse) MarshalJSON() ([]byte, error) {
 	return json.Marshal(resp)
 }
 
+func (x *InsertUserMetadataRequest) UnmarshalJSON(data []byte) error {
```

Comment:
Why do we need custom unmarshalling?
Can we define `UserMetadataValue` in the proto?

Reply:
value is []byte, without this it works with base64 encoded data. this is to give human readable interface to user. please let me know if there is alternate to handle this.

Reply:
That makes sense.
But we still expect some structure of that object?
```
if k == "value" {
	x.Value = &UserMetadataValue{
		Value: v,
	}
}
```

Reply:
synced up offline and made value to be freeform.